src/ostree.scss

```diff
@@ -77,3 +83,21 @@
 .pf-v5-c-button.pf-m-link.pf-m-inline {
     padding-top: var(--pf-v5-c-button--PaddingTop);
 }
+
+.grey-color {
+  color: var(--pf-v5-global--Color--200);
+}
+
+.danger-color {
+  color: var(--pf-v5-global--danger-color--200);
+}
+
+.warning-color {
+  color: var(--pf-v5-global--warning-color--100);
+}
```

Comment:
Why do we have these classes here?

Why not use the PF utility classes instead?

https://www.patternfly.org/utility-classes/text#status-and-state-text-colors

- grey: `pf-v5-u-color-200`
- danger: `pf-v5-u-danger-color-200`
- warning: `pf-v5-u-warning-color-100` (this is probably too light and needs to be 200 if it's text on a white background)